openproblems/tasks/label_projection/methods/knn_classifier.py

```diff
@@ -14,7 +13,6 @@
     paper_year=1967,
     code_url="https://scikit-learn.org/stable/modules/generated/"
     "sklearn.neighbors.KNeighborsClassifier.html",
-    code_version=check_version("scikit-learn"),
```

Comment:
why remove this?

Reply:
Related to my question above... decorators don't work nicely with the docker images?

Reply:
See above